File: TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target)
    {
        int [] indices = new int [2];
        for(int a = 0; a < nums.length-1; a++)
        {
            for(int b = 1; b< nums.length; b++)
            {
                if(nums[a] + nums[b] == target)
                {
                indices[0] = a;
                indices[1] = b;
                }
            }
        }
        
        return indices;
    }
}

File: twoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        answer = []
        dictionary = {}

        for index in range(len(nums)):
            nums2 = target - nums[index]
            
            if nums2 in dictionary:
                answer.append(index)
                answer.append(dictionary[nums2])

            dictionary[nums[index]] = index
        
        return answer
        